src/app/home/home.page.ts

```diff
@@ -106,20 +111,17 @@ export class HomePage {
       );
 
     return combineLatest([
-      fetchData(urls.improvements),
+      fetchDataImprovements(improvements),
       fetchData(urls.spotlight),
-      fetchData(urls.recommendation),
     ])
       .pipe(
-        map(([improvementRes, spotlightRes, recommendationRes]: any) => {
-          this.myImprovements = improvementRes.result.data;
-          this.improvementsCount = improvementRes.result.count;
+        map(([improvementRes, spotlightRes]: any) => {
+          this.myImprovements = improvementRes.result[0].data;
```

Comment:
why are we hardcoding index value here?

Reply:
in the api response we are getting the response structure like this